circle/src/deep_quotient.rs

```diff
@@ -101,9 +104,13 @@ pub fn extract_lambda<F: ComplexExtendable, EF: ExtensionField<F>>(
 
     // < v_d, v_d >
     // This formula was determined experimentally...
-    let v_d_2 = F::two().exp_u64(lde_domain.log_n as u64 - 1);
+    let v_d_2 = F::two().exp_u64(log_lde_size as u64 - 1);
```

Comment:
Nice, did not think yet about an explicit formula of `<v_H,V_H>_D`.

Reply:
Have a proof: 
Under the  `(n-1)`-fold group squaring/doubling map `\pi^{n-1}`, we have
```
D_{n+ beta} ----- \pi^{n-1} -----> D_{beta + 1},
```
where the two domains are standard position cosets. Since the vanishing poly `v_n = (pi^{n-1})_x`, i.e. the `x`-coordinate of `\pi^{n-1}`, we have
```
< v_n, v_n >_{D_{n+beta}} = 2^{n-1} * < x, x >_{D_{beta + 1}},
```
since each fiber of the projection consists of `2^{n-1}` points. Hence all we need is the value of the sum of x^2 over any standard position coset (in particular, that of size `beta + 1`).
Writing `f(x) = x^2` in FFT basis, 
``` 
x^2 = 1/ 2 * (2 x^2 - 1) + 1/2,
```
and knowing that that any non-constant basis vector sums up to zero (this is similar to an harmonic FFT), we conclude that 
```
< x, x >_{D_{beta + 1}} = 1/2 * |D_{beta + 1}| = 2^beta.
```
Altogether, `< v_n, v_n >_{D_{n+beta}} = 2^{n + beta - 1}`.

Reply:
Anyway, we should point out that this formula holds for standard position cosets.